VS_Project/Pokerapp/Pokerapp/end-game.cshtml

```diff
@@ -0,0 +1,55 @@
+﻿@using WebMatrix.Data
+
+@{
+    string connectionString = @"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Database.mdf;Integrated Security=True";
+    string provider = "System.Data.SqlClient";
+    Database db = Database.OpenConnectionString(connectionString, provider);
+
+    int roomcode = 1234567;
+
+    if (IsPost && !Request.Form["quit"].IsEmpty())
+    {
+        int endgamequery = (int)db.Execute("UPDATE Players SET Tbl_nr = NULL, Rebuy = NULL, Active = NULL WHERE Room_Code = @0", roomcode);
+
+
+        Response.Redirect("insertpagehere.cshtml");
+    }
+
+
+}
+
+
+<!DOCTYPE html>
```

Comment:
Waarom zitten hier html, head, body en doctype tags in? die staan allemaal in de _Layout.cshtml en ik heb iedereen verzocht die eruit te halen voordat je verder pusht een paar dagen geleden.